examples/single-direction-scroll/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Blockly Demo: Single-direction Scrolling</title>
+  <!--  TODO: Change back to unpkg and remove relative path before merging -->
```

Comment:
Looks like this TODO should be done now?

Reply:
done